pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -686,7 +795,7 @@ func minDuration(durations []time.Duration) time.Duration {
 		return time.Duration(0)
 	}
 
-	minDuration := time.Duration(1 * time.Hour)
+	minDuration := time.Hour
```

Comment:
Are we sure these calls are exactly equivalent?

Reply:
I believe so.
I actually caught it because of a go lint warning.
time.Hour is of Type Duration so conversion is not necessary, and since Duration is fundamentally an int64 I see no point for multiplying it by 1.